src/sidebar/templates/annotation.html

```diff
@@ -189,21 +185,19 @@
         </annotation-share-dialog>
       </span>
       <span ng-if="vm.canFlag()">
-        <button class="btn btn-clean annotation-action-btn"
-          ng-if="!vm.isFlagged()"
-          ng-click="vm.flag()"
-          ng-disabled="vm.isDeleted()"
-          aria-label="Flag"
-          h-tooltip>
-          <i class="h-icon-annotation-flag btn-icon"></i>
-        </button>
-        <button class="btn btn-clean annotation-action-btn"
-          ng-if="vm.isFlagged()"
-          ng-disabled="vm.isDeleted()"
-          aria-label="Annotation has been flagged"
-          h-tooltip>
-          <i class="h-icon-annotation-flag annotation--flagged btn-icon"></i>
-        </button>
+        <annotation-action-button
+         icon="'h-icon-annotation-flag'"
+         is-disabled="vm.isDeleted()"
+         label="'Flag'"
+         ng-if="!vm.isFlagged()"
+         on-click="vm.flag()"
+        ></annotation-action-button>
+        <annotation-action-button
+         icon="'h-icon-annotation-flag annotation--flagged'"
+         is-disabled="vm.isDeleted()"
+         label="'Annotation has been flagged'"
+         ng-if="vm.isFlagged()"
+        ></annotation-action-button>
```

Comment:
In follow up work, it might be possible to extract out a separate `<annotation-flag-button>` component that encapsulates the three `ng-if`s here and handles showing or not showing itself, and turning itself red or not. It could also have its own tests which should simplify `annotation-test.js`